docs/src/api/class-download.md

```diff
@@ -54,6 +54,12 @@ downloaded content. If [`option: acceptDownloads`] is not set, download events a
 not performed and user has no access to the downloaded files.
 :::
 
+## async method: Download.cancel
```

Comment:
Let's call it `Download._cancel` Until it's implemented in all browsers, add a note that this is an experimental API and rename it later to `Download.cancel` when it's supported in all browsers (or just remove it without breaking our API contract if we cannot implement it in other browsers).